velox/common/caching/SsdFile.cpp

```diff
@@ -119,8 +119,16 @@ SsdFile::SsdFile(const Config& config)
   filesystems::FileOptions fileOptions;
   fileOptions.shouldThrowOnFileAlreadyExists = false;
   fileOptions.bufferIo = !FLAGS_velox_ssd_odirect;
-  writeFile_ = fs_->openFileForWrite(fileName_, fileOptions);
-  readFile_ = fs_->openFileForRead(fileName_, fileOptions);
+
+  try {
```

Comment:
Can we have wrapper for file open which take cares of the metric counting? @zacw7

Reply:
I think @xiaoxmeng wants to handle all the openFile* calls in this file using a wrapper and clean-up all the try-catch blocks.